src/onedrive-reverse.js

```diff
@@ -58,7 +58,9 @@ async function reverseLookup(opts) {
     log,
   });
 
-  const segs = uri.pathname.substring(1).split('/');
+  const segs = uri.pathname.substring(1)
+    .split('/')
+    .filter((s) => s !== 'r' && s !== ':w:');
```

Comment:
Do we need special handling for Excel (`:x:`) too?

Reply:
maybe ? ... I'll add it